packages/manager/modules/nasha/src/components/partition/edit-description/edit-description.controller.js

```diff
@@ -0,0 +1,34 @@
+import { DESCRIPTION_MAX } from '../partition.constants';
+
+export default class NashaComponentsPartitionEditDescriptionController {
+  /* @ngInject */
+  constructor($translate, $http) {
+    this.$translate = $translate;
+    this.$http = $http;
+    this.model = { description: '' };
+    this.descriptionMax = DESCRIPTION_MAX;
+  }
+
+  $onInit() {
+    this.model.description = this.partition.description;
+  }
+
+  submit() {
+    const { description } = this.model;
+    const { serviceName } = this.nasha;
+    const { partitionName } = this.partition;
+
+    this.$http
```

Comment:
you have to add `return` behavior to benefit to one time-binding in your html template.
Here you return an undefined, Angular will retain the final value as long as is different from `undefined`.